app/views/declare-partnership/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>
+        {% extends "layout.html" %}
```

Comment:
If we extend layout, do we need the first part of the file? Can we stick them in `layout.html` and forget about them?

Reply:
Nope, don't need these, thought I'd got rid of them. Good spot